pkg/kafka/aclutil/acl_table.go

```diff
@@ -16,10 +16,10 @@ type permissionsRow struct {
 }
 
 // MapACLsToTableRows converts a list of ACL bindings into a formatted table for printing
-func MapACLsToTableRows(bindings []kafkainstanceclient.AclBinding, localizer localize.Localizer) []permissionsRow {
-	rows := make([]permissionsRow, len(bindings))
+func MapACLsToTableRows(bindings *[]kafkainstanceclient.AclBinding, localizer localize.Localizer) []permissionsRow {
```

Comment:
A slice wraps an array, and holds a reference to that array already, so an extra reference is unnecessary:

https://golang.org/doc/effective_go#slices